bitcoin/src/blockdata/script/owned.rs

```diff
@@ -131,7 +131,7 @@ mod tmp_pub {
         pub fn push_opcode(&mut self, data: Opcode) { self.push(data.to_u8()); }
 
         /// Adds instructions to push some arbitrary data onto the stack.
-        pub fn push_slice<T: AsRef<PushBytes>>(&mut self, data: T) {
+        pub fn push_slice(&mut self, data: impl AsRef<PushBytes>) {
```

Comment:
Losing turbofish may be actually annoying here. I question the difficulty of adjusting the macro. Isn't it just a matter of adding `$(<$($fn_gen:ident$(: $fn_gent:ty)),*>)?` ?

Side note: calling out a breaking change when you already did one in previous commit is kinda funny. :) But I guess "more breaking than expected" makes sense.